samples/highcharts/demo/combo-meteogram/demo.js

```diff
@@ -69,24 +209,31 @@ Meteogram.prototype.smoothLine = function (data) {
  * https://github.com/YR/weather-symbols
  */
 Meteogram.prototype.drawWeatherSymbols = function (chart) {
-    var meteogram = this;
 
     chart.series[0].data.forEach((point, i) => {
-        if (meteogram.resolution > 36e5 || i % 2 === 0) {
-
-            chart.renderer
-                .image(
-                    'https://cdn.jsdelivr.net/gh/YR/weather-symbols@6.0.2/dist/svg/' +
-                        meteogram.symbols[i] + '.svg',
-                    point.plotX + chart.plotLeft - 8,
-                    point.plotY + chart.plotTop - 30,
-                    30,
-                    30
-                )
-                .attr({
-                    zIndex: 5
-                })
-                .add();
+        if (this.resolution > 36e5 || i % 2 === 0) {
+
+            const [symbol, specifier] = this.symbols[i].split('_'),
+                icon = Meteogram.dictionary[symbol].symbol +
+                    ({ day: 'd', night: 'n' }[specifier] || '');
+
+            if (Meteogram.dictionary[symbol]) {
+                chart.renderer
+                    .image(
+                        'https://cdn.jsdelivr.net/gh/YR/weather-symbols' +
+                            `@6.0.2/dist/svg/${icon}.svg`,
```

Comment:
It's a good opportunity to change the `@6.0.2` version or maybe even get rid of it?
https://cdn.jsdelivr.net/gh/YR/weather-symbols/dist/svg/10.svg

Reply:
Yes, bumped to v8.0.1. I think it's a little dangerous to remove versioning though, in case they change naming, formats etc.